host/src/containers/GameSessionContainer.tsx

```diff
@@ -187,33 +232,40 @@ const GameSessionContainer = () => {
   const handleBeginQuestion = () => {
     // I'm keeping this console.log in until we figure out NOT_STARTED so we can tell there's been a change in state 
     console.log(gameSession.currentState);
+    const gameSessionId = gameSession?.id;
+    const questionId = gameSession.questions[gameSession.currentQuestionIndex].id;
+    const order = gameSession.questions[gameSession.currentQuestionIndex].order;
```

Comment:
Why one line uses `gameSession?` and not the other ones? Can also use `isNullOrDefault` and do early exist which ensures `gameSession` is not `null` and Typescript guarantees it as well

Reply:
Added the `isNullOrDefault` check above and removed the `?`